tests/serial/action-timings/ActionTimingsImplicit.cpp

```diff
@@ -63,21 +63,15 @@ BOOST_AUTO_TEST_CASE(ActionTimingsImplicit)
 
   dt = interface.initialize();
   BOOST_TEST(dt == 1.0);
-
   if (context.isNamed("SolverOne")) {
-    BOOST_TEST(action::RecorderAction::records.size() == 2);
-    BOOST_TEST(action::RecorderAction::records.at(0).timing == action::Action::WRITE_MAPPING_PRIOR);
-    BOOST_TEST(action::RecorderAction::records.at(1).timing == action::Action::WRITE_MAPPING_POST);
+    BOOST_TEST(action::RecorderAction::records.empty());
```

Comment:
Without initialization, Nothing actually happens here for the first participant.